src/services/get-env-api.js

```diff
@@ -1,8 +1,20 @@
 import store from "../store";
 import constants from "../utils/constants";
 
-export default function () {
-  return store.getters.env === "test"
-    ? constants.api.testnet
-    : constants.api.mainnet;
+const CURRENT_API_VERSION = import.meta.env.VITE_CURRENT_API_VERSION;
+
+export default function getEnvApi(version = CURRENT_API_VERSION) {
+  let apiEndpoint =
+    store.getters.env === "test"
+      ? constants.api.testnet
+      : constants.api.mainnet;
+
+  if (version === false) {
```

Comment:
can it be replaced with `!version` ?

Reply:
It will also accept `null` in that case

Reply:
I guess that will be fine because anything other than numbers greater than zero means `unversioned api` right?